packages/utilities/.env

```diff
@@ -0,0 +1 @@
+GEO_API=https://api.www.s81c.com/webmaster/dbip/
```

Comment:
@annawen1 this file shouldn't be checked in, was `.env` added to `.gitignore`?

Reply:
whoops!